src/artemis/ispyb/ispyb_dataclass.py

```diff
@@ -29,7 +29,8 @@ class IspybParams:
     )
 
     synchrotron_mode: str
-    xtal_snapshots: List[str]
+    xtal_snapshots_y: List[str]
+    xtal_snapshots_z: List[str]
```

Comment:
**nit**: It's only y/z 'cos we're assuming 0/90. Maybe better to do something like omega start and omega end?